tests/garage/theano/algos/test_algos.py

```diff
@@ -68,7 +69,7 @@
 }
 
 polopt_cases = []
-for algo in [VPG, TNPG, PPO, TRPO, CEM, CMAES, ERWR, REPS]:
+for algo in [VPG, TNPG, PPO, TRPO, CEM, ERWR, REPS]:
```

Comment:
If you want to mark CMAES flaky, you will need to split these into separate test cases. It's not that flaky, so I recommend you just leave it.

Reply:
done